DataConnectors/CohesitySecurity/AlertHttpTrigger/AlertHttpTrigger.cs

```diff
@@ -26,7 +26,7 @@ public static class AlertHttpTrigger
 
         public static long GetPreviousUnixTime()
         {
-            DateTime previousDateTime = DateTime.Now.AddDays(-30);
+            DateTime previousDateTime = DateTime.Now.AddDays(long.Parse(Environment.GetEnvironmentVariable("startDaysAgo")));
```

Comment:
Add exception handling, and in case of exception, display error in the debug output and make startDaysAgo -30